package.json

```diff
@@ -16,49 +16,29 @@
   "license": "MIT",
   "author": {
     "name": "Rui Marinho",
-    "email": "rui.marinho@seegno.com",
-    "url": "http://seegno.com"
+    "email": "ruipmarinho@gmail.com",
+    "url": "https://github.com/ruimarinho"
   },
-  "main": "./dist/index.js",
   "repository": {
     "type": "git",
     "url": "https://github.com/seegno/debugnyan.git"
   },
   "scripts": {
     "changelog": "github_changelog_generator --header-label='# Changelog' --future-release=v$npm_config_future_release && sed -i '' -e :a -e '$d;N;2,4ba' -e 'P;D' CHANGELOG.md",
-    "cover": "nyc --reporter=html --reporter=text npm test",
-    "lint": "eslint src test",
-    "prepublish": "npm run transpile",
-    "test": "NODE_ENV=test mocha $npm_package_options_mocha",
-    "transpile": "rm -rf dist/* && babel src --out-dir dist",
-    "version": "npm run changelog --future-release=$npm_package_version && npm run transpile && git add -A CHANGELOG.md dist"
+    "lint": "eslint index.js test",
+    "test": "jest --coverage",
```

Comment:
Remove `--coverage` and pass [jest options](https://facebook.github.io/jest/docs/configuration.html#content)?

In this case:

```js
{
  …,
  "collectCoverage": true,
  "testEnvironment": "node"
}
```